eclair-node-gui/src/main/scala/fr/acinq/eclair/gui/utils/CoinFormat.scala

```diff
@@ -0,0 +1,28 @@
+package fr.acinq.eclair.gui.utils
+
+import java.text.{DecimalFormat, NumberFormat}
+import java.util.Locale
+
+object CoinFormat {
+  /**
+    * Always 3 decimals, maximum of 8
+    */
+  val BTC_PATTERN = "###,##0.000#####"
```

Comment:
It looks like `BTC_PATTERN`/`BTC_FORMAT` are unused?

Reply:
It is unused for now. The code can be removed but I'm pretty sure it will be needed.